iOS/FVM/Framework/DamagedParts/DamagedPartsInitializer.swift

```diff
@@ -26,15 +25,16 @@ internal class DamagedPartsInitializer : DamagedPartsInitializerProtocol{
         self.nodeHelper = nodeHelper
         self.carModel = carModel
         self.damagePartsService = damagePartsService
-        self.initialConfiguration = initialConfiguration
+        self.initialConfiguration = initialConfiguration.replacingOccurrences(of: " ", with: "").replacingOccurrences(of: "\n", with: "")
     }
     
-    func Initialize(damagedPartsNamesToHightlight: [String]) {
-        
+    public func initialize(damagedPartsNamesToHightlight: [String]) {
         let initialDamagedParts = damagePartsService.createAndGetCollectionOfDamagedParts(json: initialConfiguration)
-        
         let nodesToHighlight = nodeHelper.getIdsOfSelection(selections: initialDamagedParts)
-        
         nodeHelper.updateDamagedPartsOnUI(damagedPartsNames: nodesToHighlight, carModel: carModel)
     }
+    
+    public func getInitialConfiguration() -> String {
+        return initialConfiguration
```

Comment:
Looks like a code smell. Why you need this method? In the context of creating DamagedPartsInitializer, you have initialConfiguration as a original string.

Reply:
Just encapsulation, I didn't want to expose this field. Changed this to public property, what do you think?

Reply:
This workflow doesn't make sense to me. We get **initialConfiguration** -> we remove whitespace -> store this string in initializer -> return to app onCancel(). In this context **DamagedPartsInitializer** looks like container for data, but also it has another responsibility - **initializingSelection**. 

Possible solutions:

1. Store **initialConfiguration** in controller (consider using MVVN in the future)
2. Create Configuration class which will store such basic informations about app.
3. Move it to **DamagedPartsService**

Reply:
Alright, I see your point. I'll move it to the Service, seems reasonable to me, as it's services purpose to return outcome. Also, I will revert extracting Initializer to private field.